lib/OpenQA/LiveHandler/Controller/LiveViewHandler.pm

```diff
@@ -197,7 +197,7 @@ sub handle_message_from_java_script {
     try {
         $json = decode_json($msg);
     }
-    catch {
+    catch ($error) {
```

Comment:
The common style is `catch ($e)` same as we use in os-autoinst. No reason to be different just for the sake of it.

Same for other occurences later on

Reply:
not for the sake to be different. 
- IMO it is more clear
- can reused in other places where $_ is stored in $error

Reply:
i will update it